### PR TITLE
Refresh user data

### PR DESCRIPTION
**Pull request by MatYouKy**
# Related issue
- User avatar/displayName not updating at FeedPost

# Changes
- Add userData to useEffect dependency

# Additional information
- none

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
     "extends": [
-      "stylelint-config-sass-guidelines",
-      "stylelint-config-prettier"
```
Comment (MateuszLisowski) [inline .stylelintrc.json:4]: why did you delete it?

File: .stylelintrc.json [hunk 2]
```diff
@@ -11,7 +10,11 @@
     ],
     "rules": {
       "at-rule-no-unknown": null,
-      "scss/at-rule-no-unknown": true
+      "scss/at-rule-no-unknown": true,
+      "max-nesting-depth": 5,
+      "resolveNestedSelectors": true,
+      "scss/dollar-variable-pattern": "^foo",
```
Comment (MateuszLisowski) [inline .stylelintrc.json:16]: what does it mean?

Why did you add these rules?

File: src/components/DropdownsContent/HeaderDropdown.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ import Brightness7Icon from '@material-ui/icons/Brightness7';
 import PropTypes from 'prop-types';
 import { useHistory } from 'react-router-dom';
 import { AuthContext } from '../../context/AuthProvider';
-import { signOut } from '../../actions/signOut';
+import { signOut } from '../../actions/signout';
```
Comment (MateuszLisowski) [inline src/components/DropdownsContent/HeaderDropdown.tsx:9]: there is no file like this

**Review by Polo11121 — APPROVED** [review]

File: .stylelintrc.json (hunk 1 above)
Comment (MatYouKy) [reply .stylelintrc.json:4]: My oversight, restoring to the previous state.

File: .stylelintrc.json (hunk 2 above)
Comment (MatYouKy) [reply .stylelintrc.json:16]: These were old settings that I didn't remove before commit. I restore to the previous state.

**Review by Polo11121 — APPROVED** [review]

**Review by MateuszLisowski — APPROVED** [review]
